### ISSUE TITLE
MNIST Optimizer

### ISSUE THREAD

**Issue by christywarden734** [pytorch/examples#736]
Changing the optimizer to optim.SGD produces all nan for the loss. 

https://github.com/pytorch/examples/blob/master/mnist/main.py#L116

**Comment by bentrevett** [comment]
If you want to switch to SGD then you should also use a lower learning rate.

**Comment by soumith** [comment]
what @bentrevett said
